src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -148,14 +147,12 @@ public void addSuperUserToOrganisation(
                 RefDataUtil.removeAllSpaces(userCreationRequest.getEmail().toLowerCase()),
                 organisation);
 
-        List<String> jurisdictionIds = userCreationRequest.getJurisdictions().stream().map(Jurisdiction::getId)
-                .collect(Collectors.toList());
 
         ProfessionalUser persistedSuperUser = professionalUserRepository.save(newProfessionalUser);
 
         List<UserAttribute> attributes
-                = userAttributeService.addUserAttributesToSuperUserWithJurisdictions(persistedSuperUser,
-                newProfessionalUser.getUserAttributes(), jurisdictionIds);
+                = userAttributeService.addUserAttributesToSuperUserWithoutJurisdictions(persistedSuperUser,
```

Comment:
Better not to keep Jurisdiction here onwards, just keep method name .addUserAttributesToSuperUser

Reply:
Done